package.json

```diff
@@ -12,10 +12,11 @@
   "typings": "build/src/index.d.ts",
   "scripts": {
     "build": "npm-run-all -p lint:tslint build:tsc -s build:webpack",
-    "build:tsc": "tsc -p test/",
+    "build:tsc": "tsc -p .",
```

Comment:
Building test files already builds the src (since the src/ tree is a subset of the test/) and isn't much slower. Can we keep building test/?